### PR TITLE
Ear 1702 force password change

### PR DESCRIPTION
**Pull request by jasehumphr**
https://jira.ons.gov.uk/browse/EAR-1702

To Test

- Spin up the Author
- Try to sign in using either a password shorter that 8 chars and then a common password eg facebook
- Check the appropriate message appears.
- Check you can still sign in with a suitable strength password.

### CODE REVIEW

File: eq-author/src/App/SignInPage/SignInForm/index.js [hunk 1]
```diff
@@ -70,6 +64,44 @@ const SignInForm = ({
...
+    setEmailNowVerified(false);
+    setPasswordExpired(false);
+    if (email === "") {
+      setErrorMessage("Enter email");
+      return;
+    } else if (password === "") {
+      setErrorMessage("Enter password");
+      return;
+    } else if (password.length < 8 && password.length !== 0) {
+      setErrorMessage("Your password has expired.");
```
Comment (farres1) [inline eq-author/src/App/SignInPage/SignInForm/index.js:86]: From the ticket's design screenshot, the "Your password has expired" error message also has a link to the "Reset your password" page after the full stop.

File: eq-author-api/docker-compose.yml [hunk 2]
```diff
@@ -21,7 +21,8 @@ services:
       - AWS_SECRET_ACCESS_KEY=dummy
       - NODE_ENV=development
       - RUNNER_SESSION_URL=http://localhost:5000/session?token=
-      - PUBLISHER_URL=http://host.docker.internal:9000/publish/
+      - PUBLISHER_URL=http://host.docker.internal:4000/convert/
+      - CONVERSION_URL=http://host.docker.internal:9000/publish/
       - SURVEY_REGISTER_URL=http://host.docker.internal:8080/submit
       - ENABLE_IMPORT=true
       - JAEGER_SERVICE_NAME=eq_author_api
```
Comment (farres1) [inline eq-author-api/docker-compose.yml:28]: The gcp feature flag will also need to be added to the docker-compose yml file for v3 to run

**Review by farres1 — CHANGES_REQUESTED** [review]
The password change is working really well!
I've added two comments with small changes, a design change from the prototype and a change to fix v3

File: eq-author/src/App/SignInPage/SignInForm/index.js (hunk 1 above)
Comment (martyncolmer) [reply eq-author/src/App/SignInPage/SignInForm/index.js:86]: Agreed with Joe to change the bottom message instead

File: eq-author-api/docker-compose.yml (hunk 2 above)
Comment (martyncolmer) [reply eq-author-api/docker-compose.yml:28]: Done

**Review by farres1 — APPROVED** [review]

**Review by oms09 — APPROVED** [review]

**Pull request merged**